requirements.in

```diff
@@ -35,5 +36,5 @@ wagtail-metadata
 whitenoise
 psycopg2-binary
 sentry-sdk
-git+https://github.com/MozillaFoundation/wagtail-footnotes.git@localized-footnotes
+git+https://github.com/kevinhowbrook/wagtail-footnotes-moz-foundation.git@localized-footnotes
```

Comment:
**todo**

This will need testing and updating on the actual fork, I've forked it again for efficiency here

Reply:
I left one question on that footnotes PR

Reply:
(for clarity) 

> Is the translatability maybe something that is worth contributing back to the original package?

@tbrlpld - I'm unsure, given it wasn't originally contributed I don't know if I can answer that. But what I can say is that all we need is the wagtail requirement altering for our wagtail upgrade. I would suggest that yes, this is something we should merge into footnotes - but it will need more work as I think the migrations are out of sync with the fork and it's likely to hold up the upgrade for a bit.

Reply:
re-targeted in 2228dc9fab58fa956b2c4c553a38168aa50cd95b